akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
Oh, that's interesting, thanks for the pointer. Would you like me to copy the same design ?

About case classes, is what you mention related to this: https://issues.scala-lang.org/browse/SI-9627 ?